src/ts/components/subtitleoverlay.ts

```diff
@@ -36,13 +46,18 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
     player.addEventHandler(player.EVENT.ON_CUE_ENTER, (event: SubtitleCueEvent) => {
       let labelToAdd = subtitleManager.cueEnter(event);
 
-      if (this.previewSubtitleActive) {
-        this.removeComponent(this.previewSubtitle);
+      if (this.isCEA608) {
```

Comment:
Remove all CEA-608 stuff from `configure` and handle it only in `configureCea608Captions`. Adding of the label can stay here like it was. The other handler in `configureCea608Captions` should just get the existing cue(s) with `getCues` and apply the style.